packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -459,11 +459,16 @@ const extractConfigurationsFromInputs = async () => {
 		return;
 	}
 
-	if (documentIds.value) {
+	const configExtractionDocumentIds = modelConfigurations.value
+		.map((config) => config.extractionDocumentId)
+		.filter((id) => id);
+
+	const newDocumentIds = difference(documentIds.value, configExtractionDocumentIds);
+	if (newDocumentIds.length) {
```

Comment:
We are using `_.isEmpty()` everywhere in the app.
```suggestion
	if (!isEmpty(newDocumentIds)) {
```